### PR TITLE
Depth map completed

### PR DESCRIPTION
**Pull request by mansi75**
KIndly merge the pull request

### CODE REVIEW

**Review by geekquad — CHANGES_REQUESTED** [review]
Accepting only .ipython notebooks.

File: Riddler-Calvard/riddler_calvard.ipynb [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+      "cell_type": "code",
+      "metadata": {
+        "colab": {
+          "base_uri": "https://localhost:8080/"
+        },
+        "id": "L5R-tQX13TpI",
+        "outputId": "ae182f00-cc5c-418f-e94b-9ce87ba45993"
+      },
+      "source": [
+        "!pip install mahotas\n"
```
Comment (geekquad) [inline Riddler-Calvard/riddler_calvard.ipynb:29]: ```mahotas``` not allowed. 
Please red the Contributing Guidelines.

Comment (mansi75) [reply Riddler-Calvard/riddler_calvard.ipynb:29]: @geekquad Riddler-calvard without mahotas?

**Comment by mansi75** [comment]
@geekquad Changes made in depth notebook

File: Riddler-Calvard/riddler_calvard.ipynb (hunk 1 above)
Comment (geekquad) [reply Riddler-Calvard/riddler_calvard.ipynb:29]: Our focus is to have various feature implementations of OpenCV in Python.

**Review by geekquad — CHANGES_REQUESTED** [review]
Please add the sample input image in the notebook as well.

**Comment by mansi75** [comment]
@geekquad Kindly check again, I have done it

**Review by geekquad — APPROVED** [review]

**Review by geekquad — COMMENTED** [review]
1. Depth Map is okay.
2. I cannot see any Image Processing technique being used in Geometric shapes.
3. If mahotas is not being used, please omit that from the codebase.
4. I cannot merge specific files from the PR as such, make different branches for each of the issues.

**Comment by mansi75** [comment]
1. In Geometric shapes, I have used OpenCV methods cv2.line, cv2.rectangle, cv2.circle etc to draw various shapes on images. Kindly check I have updated it too. 
2. I have removed riddler calvard from the repo since ridler calvard can not be used without mahatos.

**Comment by geekquad** [comment]
Hey @mansi75, reverted you back. 
Kindly check the discord server.

**Comment by mansi75** [comment]
@geekquad 
Kindly check again

**Review by geekquad — APPROVED** [review]
Heu @mansi75, good work!

**Pull request merged**